packages/@navikt/side-menu/src/MenuLink.tsx

```diff
@@ -28,17 +29,42 @@ const MenuLink = ({ label, active, onClick, index, iconSrc, iconAltText }: MenuL
         }
     );
 
+    const containerClassnames = classnames(menuLinkCls.block, {
+        [menuLinkCls.element('arrow-theme')]: arrowTheme,
+    });
+
+    const getLabel = () => {
```

Comment:
Bør sikkert lage en egen komponent av dette. Det fins også en generell typografi-komponent i `nav-frontend-typografi` som kan brukes for å gjøre koden logikken her litt enklere (se [her](https://github.com/navikt/nav-frontend-moduler/blob/master/packages/node_modules/nav-frontend-typografi/src/index.tsx)).

Så f.eks:
```
const MenuLinkLabel = ({ shouldBoldenText, children }) => (
    <Typografi type={shouldBoldenText ? 'element' : 'normaltekst'}>
        {children}
    </Typografi>
);

<MenuLinkLabel shouldBoldenText={theme === Theme.Arrow && isActive} />
```